builder/googlecompute/step_check_existing_image.go

```diff
@@ -17,15 +17,18 @@ func (s *StepCheckExistingImage) Run(state multistep.StateBag) multistep.StepAct
 	d := state.Get("driver").(Driver)
 	ui := state.Get("ui").(packer.Ui)
 
-	ui.Say("Checking image does not exist...")
-	exists := d.ImageExists(c.ImageName)
-	if exists {
-		err := fmt.Errorf("Image %s already exists", c.ImageName)
+	if !c.PackerForce {
+		ui.Say("Checking image does not exist...")
```

Comment:
I think we should always say if we're checking for the image even if we're not in force mode